### ISSUE TITLE
[Bug] TypeError: Cannot read property 'focus' of null

### ISSUE THREAD

**Issue by azhavoro** [openvinotoolkit/cvat#2750]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

```js
TypeError: Cannot read property 'focus' of null
    at https://cvat.org/cvat-ui.934ea8f5345500f73272.min.js:397:1060933
```

### Steps to Reproduce (for bugs)

1. N/A

### Your Environment

- Git hash commit (`git log -1`): 3ced77ccc6065ab57c64a50fba81bc16573d5d40
- Docker version `docker version` (e.g. Docker 17.0.05):
- Are you using Docker Swarm or Kubernetes?
- Operating System and version (e.g. Linux, Windows, MacOS):
- Code example or link to GitHub repo or gist to reproduce problem:
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container</summary>
    </details>

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

**Comment by bsekachev** [comment]
Please, look at this bug. Need to find a reason and fix it before release 1.3.0-alpha. 
You can use Kibana analytics on cvat.org to find out user scenarios

**Comment by bsekachev** [comment]
@vnishukov 

Please, look at this bug. Need to find a reason and fix it before release 1.3.0-alpha.
You can use Kibana analytics on cvat.org to find out user scenarios

**Comment by bsekachev** [comment]
Latest time it was reproduced 18th of March. I will close the issue for now.
